webapp/channels/src/components/admin_console/access_control/editors/cel_editor/editor.scss

```diff
@@ -0,0 +1,257 @@
+        flex-grow: 0;
+        padding: 12px;
+        border: 1px solid var(--button-bg);
+        border-radius: 4px 4px 0 0;
+        font-family: monospace;
+        resize: vertical;
+        transition: border-color 0.15s ease;
+
+        &:focus {
+            outline: none;
+        }
+
+        &::placeholder {
+            color: rgba(var(--center-channel-color-rgb), 0.56);
+        }
+
+        .policyEditor {
+            outline: none;
+        }
+
+        .cel-editor__container[data-status-color='var(--error-text)'] & {
+            border-color: var(--error-text);
+        }
+
+        .cel-editor__container[data-status-color='var(--online-indicator)'] & {
+            border-color: var(--online-indicator);
+        }
+
+        .cel-editor__container[data-status-color='var(--button-bg)'] & {
+            border-color: var(--button-bg);
+        }
+    }
+
+    &__footer {
+        display: flex;
+        align-items: flex-start;
+        justify-content: space-between;
+        margin-top: 8px;
+
+        .help-text-container {
+            flex: 1;
+            padding-right: 16px;
+            color: var(--center-channel-color-72);
+            font-size: 12px;
+        }
+    }
+
+    &__actions {
+        display: flex;
+        align-items: center;
+    }
+
+
+
+    &__loading {
+        display: flex;
+        align-items: center;
+
+        i {
+            margin-right: 8px;
+        }
+    }
+  
+    &__status-bar {
+        display: flex;
+        min-height: 24px;
+        align-items: center;
+        justify-content: space-between;
+        padding: 0 8px;
+        border-radius: 0 0 4px 4px;
+        color: var(--button-color);
+        font-family: monospace;
+        font-size: 12px;
+        transition: background-color 0.15s ease;
+        
+        &:not([data-validation-state="validated"]):not([data-validation-state="error"]):not([data-validation-state="validating"]) {
+            cursor: pointer;
+            
+            &:hover {
+                background: linear-gradient(0deg, rgba(0, 0, 0, 0.08), rgba(0, 0, 0, 0.08)), var(--button-bg);
+            }
+            
+            &:active {
+                background: linear-gradient(0deg, rgba(0, 0, 0, 0.16), rgba(0, 0, 0, 0.16)), var(--button-bg);
+            }
+        }
+    }
+
+    &__error-message {
+        display: flex;
+        align-items: center;
+
+        .icon {
+            margin-right: 4px;
+            font-size: 14px;
+        }
+    }
+
+    &__error {
+        display: flex;
+        align-items: center;
+        color: var(--button-color);
+    }
+
+    &__status-message {
+        display: flex;
+        align-items: center;
+
+        .icon {
+            margin-right: 4px;
+            font-size: 14px;
+
+            &.icon-refresh {
+                cursor: pointer;
+                &:hover {
+                    opacity: 0.8;
+                }
+            }
+        }
+    }
+
+    &__inline-validate-btn {
```

Comment:
(Post-merge) Some of this could probably be done by adding the `btn-link` class to this element.

There's also probably other places I've missed mentioning that in the other CSS files as well